Hackney.Shared.HousingSearch/Gateways/Models/Processes/QueryableProcess.cs

```diff
@@ -1,22 +1,47 @@
+using Hackney.Shared.HousingSearch.Domain.Process;
 using Nest;
+using System;
 using System.Collections.Generic;
+using System.Linq;
 
 namespace Hackney.Shared.HousingSearch.Gateways.Models.Processes
 {
     public class QueryableProcess
     {
+        public Process Create()
+        {
+            var patchAssignment = PatchAssignment == null
+                ? new PatchAssignment() :
+                Domain.Process.PatchAssignment.Create(
+                    PatchAssignment.PatchId,
+                    PatchAssignment.PatchName,
+                    PatchAssignment.ResponsibleEntityId,
+                    PatchAssignment.ResponsibleName);
+
+            var relatedEntities = RelatedEntities == null
+                ? new List<RelatedEntity>() :
+                RelatedEntities.Select(x => RelatedEntity.Create(
+                                                       x.Id,
+                                                       x.TargetType,
+                                                       x.SubType,
+                                                       x?.Description)).ToList();
+
+
+
+            return Process.Create(Id, TargetId, TargetType, relatedEntities, ProcessName, patchAssignment, State);
+        }
 
         [Text(Name = "id")]
-        public string Id { get; set; }
+        public Guid Id { get; set; }
```

Comment:
The 'queryable' entities all need to have string types to match the other queryable entities (e.g. queryable asset, etc.)